cmd/estuary-shuttle/main.go

```diff
@@ -232,6 +253,14 @@ func main() {
 			return err
 		}
 
+		if cctx.Bool("jaeger-tracing") {
```

Comment:
allows estuary/shuttles to send traces to a jaeger endpoint. Disabled by default.